File: c_Programming/Assingements/Assignment 13/searcharray.c
#include<stdio.h>
#include<stdlib.h>
void main()
{
	int *arr;
    int size=10;
    arr=(int*)malloc(size*sizeof(int));
    if(arr==NULL)
    {
    	printf("memory allocation failed");
    	return;
	}

    printf("Enter 10 elements:\n");
    for(int i=0; i<size; i++)
    {
        scanf("%d",&arr[i]);
    }
    printf("Array = [ ");
    for(int i=0; i<size; i++)
    {
        printf("%d ,",arr[i]); 
    }
    printf("\b]\n");
	int ele;
	int flag=-1;
	printf("Enter a element :");
	scanf("%d",&ele);
	
	for(int i=0; i<size; i++)
	{
		if(arr[i]==ele)
		{
			flag=i;
			break;
		}
	}
	if(flag !=-1)
	printf("found");
	else 
	printf("not found");
	free(arr);
}

File: Java_Module/Assignment/Assignment6/ArrayProgram/src/SearchArray.java
import java.util.Scanner;

class SearchArray {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        int[] arr = new int[10];

        // Input
        System.out.println("Enter 10 Elements:");

        for (int i = 0; i < arr.length; i++) {
            arr[i] = sc.nextInt();
        }

        // Display Array
        System.out.print("Array = [ ");

        for (int i = 0; i < arr.length; i++) {
            System.out.print(arr[i] + " ");
        }

        System.out.println("]");

        // Search Element
        System.out.print("Enter an Element to Search: ");
        int ele = sc.nextInt();

        int flag = 0;

        for (int i = 0; i < arr.length; i++) {

            if (arr[i] == ele) {
                flag = 1;
                break;
            }
        }

        if (flag == 1) {
            System.out.println("Element Found");
        } else {
            System.out.println("Element Not Found");
        }

        sc.close();
    }
}
